src/utils/axiosWithHeaders.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const axiosWithHeader = () => {
+  const tokenFromBrowser = localStorage.getItem('tokenTiemeNdo');
+
+  const token = tokenFromBrowser ? tokenFromBrowser : false;
```

Comment:
Not sure what effect this have. If there is no token in localStorage, the value will be `undefined`, which will fail the request anyway